frontend/packages/kubevirt-plugin/src/plugin.tsx

```diff
@@ -494,7 +494,7 @@ const plugin: Plugin<ConsumedExtensions> = [
   {
     type: 'PVCCreateProp',
     properties: {
-      label: 'New with Data Upload',
+      label: 'With Data upload form',
```

Comment:
the BZ say "With form and Data upload" ?

Reply:
yes and I'm asking from Matt to rethink it :)